src/quo2/components/inputs/address_input/style.cljs

```diff
@@ -0,0 +1,40 @@
+(ns quo2.components.inputs.address-input.style
+  (:require [quo2.foundations.colors :as colors]
+            [quo2.components.markdown.text :as text]
+            [react-native.platform :as platform]))
+
+(def container
+  {:padding-horizontal 20
+   :padding-top        8
+   :padding-bottom     16
+   :height             48
+   :flex-direction     :row
+   :align-items        :flex-start})
```

Comment:
nit: maybe it's good in advance to add a container-style prop here and merge it. i.e
```
(defn container [constainer-style]
  (merge {:padding-horizontal 20
   :padding-top        8
   :padding-bottom     16
   :height             48
   :flex-direction     :row
   :align-items        :flex-start} 
   container-style))
   ```